Zend/zend_partial.c

```diff
@@ -0,0 +1,930 @@
+/*
+   +----------------------------------------------------------------------+
+   | Zend Engine                                                          |
+   +----------------------------------------------------------------------+
+   | Copyright (c) Zend Technologies Ltd. (http://www.zend.com)           |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 2.00 of the Zend license,     |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | http://www.zend.com/license/2_00.txt.                                |
+   | If you did not receive a copy of the Zend license and are unable to  |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@zend.com so we can mail you a copy immediately.              |
+   +----------------------------------------------------------------------+
+   | Authors: krakjoe <krakjoe@php.net>                                   |
+   +----------------------------------------------------------------------+
+*/
+#include "zend.h"
+#include "zend_API.h"
+#include "zend_interfaces.h"
+#include "zend_closures.h"
+#include "zend_partial.h"
+
+typedef struct _zend_partial {
```

Comment:
Having a comment here on what the three different zend_functions are may be helpful.

Reply:
In 88aa29c668293b0b2ae804716ffc3e6e641b0aaf, maybe can be improved.